docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -219,6 +219,41 @@ quarkus.mongodb.connection-string = mongodb://localhost:27017
 
 If you need more configuration properties, there is a full list at the end of this guide.
 
+== Multiple MongoDB Connections
```

Comment:
Wondering if _connection_ is the right word? We are injecting clients and not connections.

Reply:
I switch from connections to clients...